### PR TITLE
DX-1301 TT-10805 Update TIB Install with Helm Chart instruction

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

DX-1301

---

### Preview Link

<br>

[Preview](https://deploy-preview-4523--tyk-docs.netlify.app/docs/nightly/tyk-identity-broker/getting-started/)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-identity-broker/getting-started.md [hunk 1]
```diff
@@ -57,13 +57,7 @@ You can install via [packages](https://packagecloud.io/tyk/tyk-identity-broker/i
...
 ### Via Helm Chart for Kubernetes
 
-Once you have installed the Gateway and Dashboard you can configure TIB by adding its configuration environment variables under the `tib.extraEnvs` section and providing the `profiles.json` as a ConfigMap - see below. See our [TIB GitHub repo](https://github.com/TykTechnologies/tyk-identity-broker#how-to-configure-tib). Once you complete your modifications you can run the following command from the root of the repository to update your helm chart.
-
-```helm
-helm upgrade tyk-pro ./tyk-pro -n tyk
-```
-
-This chart implies there's a ConfigMap with a `profiles.json` definition in it. First create a ConfigMap in your cluster, and then use `tib.configMap.profiles` value to reference the name of this ConfigMap (tyk-tib-profiles-conf by default).
+[Tyk Helm Chart]({{<ref "product-stack/tyk-charts/overview">}}) does not support installing TIB as separate application. If you want to enable embedded TIB in Dashboard, you can do so by updating `dashboard.tib.enabled` to `true`.
```
Comment (rewsmith) [inline tyk-docs/content/tyk-identity-broker/getting-started.md]: Shouldn't it be `tyk-dashboard.tib.enabled` that needs to be updated?

Comment (carolwkc) [reply tyk-docs/content/tyk-identity-broker/getting-started.md]: Yes.. I've corrected it in https://github.com/TykTechnologies/tyk-docs/pull/4523/commits/8aee4013fbcf5c20b4f358f9faba35362e431c76 and updated docs for both tyk-dashboard and tyk-stack or tyk-control-plane case.

**Review by rewsmith — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]
PR LGTM @caroltyk Please can you label the PR to specify which version(s) you would like the docs released to, e.g now, 5.2, 5.1 etc?

**Comment by carolwkc** [comment]
@dcs3spp it can be released now.

**Comment by dcs3spp** [comment]
Thanks @caroltyk Have released to 5.3

**Pull request merged**